components/03-modules/gallery/_gallery.scss

```diff
@@ -1,20 +1,25 @@
-$gallery__icon-arrow-color         : $gray !default;
-$gallery__icon-arrow-width         : 12px !default;
-$gallery__label-offset-top         : $spacer--medium !default;
-$gallery__label-offset-left        : $spacer !default;
-$gallery__nav-vertical-margin-right: 40px !default;
-$gallery__stage-margin-bottom      : $spacer--medium !default;
-$gallery__thumb-arrow-height       : 48px !default;
-$gallery__thumb-arrow-width        : 48px !default;
-$gallery__thumb-border             : none !default;
-$gallery__thumb-border--active     : 1px solid $green !default;
-$gallery__thumb-spacing            : $spacer--medium !default;
-$gallery__thumb-width              : 48px !default;
-$gallery__thumb-width--large       : 80px !default;
+$gallery__icon-arrow-color                 : $gray !default;
+$gallery__icon-arrow-width                 : 12px !default;
+$gallery__label-offset-top                 : $spacer--medium !default;
+$gallery__label-offset-left                : $spacer !default;
+$gallery__nav-vertical-height              : 80px !default;
+$gallery__nav-vertical-margin-right--medium: $spacer--medium !default;
```

Comment:
Uue `@` instead of `--medum`

Reply:
sure. forget to check other modules to update to our new naming convention